src/editor/serialize.ts

```diff
@@ -48,7 +48,9 @@ export function mdSerialize(model: EditorModel): string {
             case Type.UserPill:
                 return (
                     html +
-                    `[${part.text.replace(/[[\\\]]/g, (c) => "\\" + c)}](${makeGenericPermalink(part.resourceId)})`
+                    `[${part.text.replace(/[[\\\]]/g, (c) => "\\" + c).replace(/\n/g, "<br/>")}](${makeGenericPermalink(
+                        part.resourceId,
+                    )})`
```

Comment:
this is getting a bit hard to read. Can we (a) factor out the part generating the `[...]` content to a local variable; (b) get a comment explaining what it does?

I think it's escaping `[`, `\` and `]` with a `\`, and replacing line break characters with `<br/>` ?